content/10.diagnostics.md

```diff
@@ -28,6 +28,26 @@ In this case the test was designed to detect sarbecoviruses, of which SARS-CoV-2
 Considering that no other sarbecoviruses are currently known to infect humans, a positive test indicates that the patient is infected with SARS-CoV-2.
 However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.
 
+##### dPCR
+
+Digital PCR (dPCR) is a new generation of PCR technologies where a sample is partitioned into thousands of compartments, such as nanodroplets or nanowells, and a PCR reaction takes place in each compartment.
+This leads to a digital read-out where each partition is either positive or negative for the nucleic acid being tested for.
+While dPCR equipment is not as common yet as that for RT-PCR, dPCR generally has higher sensitivity than can be achieved with other PCR technologies, while maintaining high specificity [@doi:10.3390/s18041271].
+High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
+Suo et al. performed a double-blind evaluation of droplet dPCR (ddPCR) for SARS-CoV-2 detection on 57 samples--43 samples from suspected positive patients, and 14 from supposed convalescents--that all tested negative for SARS-CoV-2 using RT-PCR.
+Despite the initial negative results, 33 out of 35 (94.3%) patients and 9 out of 14 (64.2%) supposed convalescents were later clinically confirmed positive.
+All of these individuals tested positive using ddPCR.
+Two symptomatic patients tested negative with both RT-PCR and ddPCR, but were later clinically diagnosed positive, and 5 of the 14 suspected convalescents tested negative by ddPCR [@doi:10.1101/2020.02.29.20029439].
+While not a complete head-to-head comparison to RT-PCR in all aspects-- e.g. no samples testing positive using RT-PCR were evaluated by ddPCR-- the study shows the potential of dPCR for viral detection in highly diluted samples.
+In a second study, Dong et al. [@doi:10.1101/2020.03.14.20036129] compared qRT-PCR and ddPCR on 194 samples, including 103 samples from suspected patients, 75 from contacts and close contacts, and 16 from suspected convalescents.
+Of the 103 suspected patient samples, 29 were reported as positive, 25 as negative, and 49 as suspected by qRT-PCR; all patients were later confirmed to be SARS-CoV-2 positive.
+Among qRT-PCR negative or suspected samples, in total 61 samples (17 negative and 44 suspected) were later confirmed to be positive by ddPCR, improving the overall detection rate among these patients from 28.2% to 87.4%.
+Of 75 patient samples from contacts and close contacts, 48 were tested negative with both methods and patients indeed remained healthy.
+Within the remaining 27 patient samples, 10 tested positive, 1 negative, 16 suspect by qRT-PCR.
+15 out of 16 suspect samples and the negative test result were overturned by ddPCR, decreasing the rate of suspect cases from 21% to 1%.
+Importantly, all samples tested positive using qRT-PCR were also tested positive using ddPCR.
```

Comment:
```suggestion
Among the 16 convalescent patients, qRT-PCR identified 12 as positive, 3 as suspect, and 1 as negative, but RT-dPCR identified all 16 as positive.
This evidence further indicates that the lower limit of detection made possible by ddPCR may be useful for identifying when COVID-19 patients are cleared of the virus.
Importantly, all samples tested positive using qRT-PCR were also tested positive using ddPCR.
```
Is it worth it to mention what happened to these samples? I thought it might be worth including, so I put together a brief explanation.

Reply:
What I try to say here is that all samples across the study that were positive by qRT-PCR were also verified as positive using dPCR; in contrast to the other study where only qRT-PCR negative samples were tested. I'll reword.

Reply:
Dear @cbrueffer,

If I can make a comment I would be a bit more specific on the experimental set up used to screen patients, i.e. genes that they used to screen patients (N, ORF1ab and E) and genes that give more sensitivity compared to others (it looks like if you use probes against the N gene it works better in reference 71). Also reference 72 ( Dong et al. ) analyse the sensitivity of one step reverse transcription-dPCR compared to RT-qPCR, worth to mention? Also detection limits for RT-dPCR in reference 72 are provided (2 copies/reaction?), worth to mention it as well? See if you and @rando2 think these points are worth to mention.

Best regards

Reply:
Thanks for the comments @aadattoli ! I'll work on incorporating your and @rando2's feedback; I've been busy was other work the last few days so please excuse the slow pace.

Reply:
> Thanks for the comments @aadattoli ! I'll work on incorporating your and @rando2's feedback; I've been busy was other work the last few days so please excuse the slow pace.

Hi @cbrueffer just tag me when this is done and I'll review one last time and merge :) I think it's a great section!

Reply:
moved to #333